test/gluehbirne/brainstormings_test.exs

```diff
@@ -92,4 +92,20 @@ defmodule Mindwendel.BrainstormingsTest do
       assert count == 1
     end
   end
+
+  describe "delete_likes" do
+    test "deletes a like", %{idea: idea, user: user} do
+      # add a like first:
+      Brainstormings.add_like(idea.id, user.id)
```

Comment:
Suggestion: you can create during setup with a factory, the method should be tested separately